src/mon/MonCommands.h

```diff
@@ -527,9 +527,10 @@ COMMAND("osd crush add " \
 	"add or update crushmap position and weight for <name> with <weight> and location <args>", \
 	"osd", "rw", "cli,rest")
 COMMAND("osd crush set-device-class " \
-	"name=id,type=CephOsdName " \
-	"name=class,type=CephString ", \
-	"set the <class> of the device <name>", \
+        "name=class,type=CephString " \
+	"name=ids,type=CephString,n=N", \
+	"set the <class> of the osd(s) <id> [<id>...]," \
+        "or use <all|any|*> to set all.", \
```

Comment:
> let's drop this commit?

@liewegas The failure is expected. And I have told you in https://github.com/ceph/ceph/pull/16283.
Can we fix the test script instead? It is annoying if we have to manually set the class of a large host/rank or something alike.
With this change we can do something like ceph osd crush set-device-class hdd 'ceph osd ls-tree host'